examples/solvers/pdhg_denoising_ROF_algorithm_comparison.py

```diff
@@ -0,0 +1,124 @@
+"""Total variation denoising using the primal-dual hybrid gradient algorithm.
```

Comment:
It also shows how to compare algorithms with ODL. Thereby this add some features that other examples don't have.